NEWS.md

```diff
@@ -1,5 +1,30 @@
 MXNet Change Log
 ================
+## 0.12.0
+### - New Features - Sparse Tensor Support
+  - Added comprehensive support for sparse matrices. See help on `mx.sym.sparse` and `mx.nd.sparse` for more info.
+  - For more information see [full release notes](https://cwiki.apache.org/confluence/display/MXNET/MXNet+0.12.0+Release+Notes)
+### - New Features - Autograd and Gluon
+  - New loss functions added - `SigmoidBinaryCrossEntropyLoss`, `CTCLoss`, `HuberLoss`, `HingeLoss`, `SquaredHingeLoss`, `LogisticLoss`, `TripletLoss`
+  - `gluon.Trainer` now allows reading and setting learning rate with `trainer.learning_rate` property.
+  - Added `mx.autograd.grad` and experimental second order gradient support (though most operators don't support second order gradient yet)
+  - Added `ConvLSTM` etc to `gluon.contrib`
+  - Autograd now supports cross-device graphs. Use `x.copyto(mx.gpu(i))` and `x.copyto(mx.cpu())` to do computation on multiple devices.
+### - Other New Features
```

Comment:
Also list `limited support for fancy indexing, full support coming soon in next release. Checkout master to get a preview`

Reply:
Done